src/webpack.dev.config.js

```diff
@@ -29,6 +29,19 @@ const config = deepAssign({
 
 }, options.webpack);
 
+// Ensure dev loaders are used.
+config.module.loaders = config.module.loaders.map(loader => {
+  if (loader.test.test('.jsx')) {
```

Comment:
I believe it is safer to evaluate the loader entry instead of test.

Maybe `loader.loader.test('babel')` ?